### ISSUE TITLE
large INT handling issues

### ISSUE THREAD

**Issue by pedigree** [RedBeardLab/rediSQL#31]
Results seem to return -1 when a large bit should be returned

```
192.168.0.45:6379> flushall
OK
192.168.0.45:6379> REDISQL.CREATE_DB db
OK
192.168.0.45:6379> redisql.exec db "CREATE TABLE ip_to_asn(start INT8, end INT8, asn int, hosts int8)"
1) DONE
2) (integer) 0
192.168.0.45:6379> redisql.exec db "insert into ip_to_asn values (2883484276228096000, 2883484280523063295, 265030, 4294967295)"
1) DONE
2) (integer) 1
192.168.0.45:6379> redisql.exec db "select * from ip_to_asn where 2883484276228096000 between start and end order by hosts desc limit 1"
```
result
```
1) 1) (integer) 0
   2) (integer) -1
   3) (integer) 265030
   4) (integer) -1
```
expected
```
1) 1) (integer) 2883484276228096000
   2) (integer) 2883484280523063295
   3) (integer) 265030
   4) (integer) 4294967295
```
changing the INT8 to/from INT doesn't appear to make any difference

2883484276228096000 > 9223372036854776000 (2^63 - the signed max of a 64bit integer)

Running Redis 4.0.11 on Ubuntu 18.04 x64, with your static 0.70 binary

```
# Server
redis_version:999.999.999
redis_git_sha1:f54cde3a
redis_git_dirty:0
redis_build_id:269a808b2b47051f
redis_mode:standalone
os:Linux 4.4.0-134-generic x86_64
arch_bits:64
multiplexing_api:epoll
atomicvar_api:atomic-builtin
gcc_version:5.4.0
process_id:30150
run_id:001157c888ccf1ec2ddbc30e206322aa0c04b88a
tcp_port:6379
uptime_in_seconds:1770
uptime_in_days:0
hz:10
configured_hz:10
lru_clock:12764423
executable:/home/gateway/redis/src/./redis-server
config_file:/home/gateway/redis/redis.conf```

**Comment by siscia** [comment]
Thanks so much for your report, extremely detailed, easy to reproduce and useful.

I create #32 that should address your issue.

I am creating now the several releases and I am going to release the version 0.7.1 as soon as possible.

Please, if you see any other issue or bug let me know here.

Since you have helped to find bugs, feel free to contact me and I can provide the PRO version to you free of charges.

**Comment by siscia** [comment]
I released the v0.7.1 which should include the fix.

The diff is here: https://github.com/RedBeardLab/rediSQL/pull/32/files

Please, if everything works now as expected close this issues!

Thanks again!

**Comment by pedigree** [comment]
Works perfect, thanks for the quick.   Now I just need to wait for Redis and Sqlite to support int128.  I'll email you about testing a Pro version
